suites/apis/dataUploadTest.js

```diff
@@ -266,7 +267,75 @@ Scenario('Upload the same file twice @dataUpload', async (sheepdog, indexd, node
   );
 }).retry(2);
 
-BeforeSuite(async (dataClient, fence, users, sheepdog, indexd, files) => {
+/**
+ * Use fence's multipart upload endpoints to upload a large data file (>5MB)
+ */
+Scenario('Multipart upload', async (files, users, fence, indexd, dataUpload) => {
```

Comment:
It would be great if you have some tests for failure cases such as wrong etag when completion?